app/src/main/java/ch/epfl/favo/view/tabs/addFavor/FavorRequestView.java

```diff
@@ -207,8 +208,10 @@ private void confirmUpdatedFavor() {
     // update lists
     updateMainActivityLists(true);
     updateViewFromStatus(getView());
-    // TODO: add db call
     showSnackbar(getString(R.string.favor_edit_success_msg));
+
+    // DB call to update Favor details
+    FavorUtil.getSingleInstance().postFavor(currentFavor);
```

Comment:
I think  updateFavor() should be called here

Reply:
The problem with updateFavor is that we need to create a hashmap of the modified fields, and in this case instead of trying to find the fields that are modified which will be messy we just simply do a post, the postFavor method will automatically override the fields of the current favor if it already exists in the database.